src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReadThreadingAssemblerArgumentCollection.java

```diff
@@ -52,6 +52,14 @@
     @Argument(fullName="padding-around-snps", doc = "Include at least this many bases around an event for calling snps", optional = true)
     public int snpPadding = 20;
 
+    /**
+     * This argument is meant for debugging. Output a -L parsable summary file with all of the boundaries and summary features of each
+     * region the assembly engine assembled.
+     */
+    @Hidden
+    @Argument(fullName="active-region-out", doc="Write debug interval list file containing summary info about each active region", optional = true)
```

Comment:
Here and above some places refer to an active region and some to an assembly region (which is a padded active region).  We need to be precise.